UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -400,6 +480,16 @@ public void ProcessAdminEnableRequest(string admin, string userToPromote)
 		}
 	}
 
+	public void AddToAdminLog(string log)
```

Comment:
Why is a logging / chat function and log file management in PlayerList? I think logic related to admin logging should be somewhere else. PlayerList is for dealing with the list of players on the server.

Reply:
Yeah ive moved it all to its own script, will need to be placed on a manager though